packages/terra-compact-interactive-list/src/subcomponents/Cell.jsx

```diff
@@ -112,10 +122,14 @@ const Cell = (props) => {
     }
   };
 
-  const handleMouseDown = () => {
+  const handleMouseDown = (event) => {
     setFocusedCell({ rowId, columnId: id });
     if (isSelectableCell && onCellSelect) {
-      onCellSelect({ rowId, columnId: id });
+      event.preventDefault();
```

Comment:
Why do we need to prevent the default behavior of a mouse down event?

Reply:
I added that line after encountering an issue when the featured icon was placed inside of a cell. Even though the icon wasn't an interactive element, per click on the icon the cell would lose the focus. By adding `event.preventDefault()` that problem resolved. As this `event.preventDefault()` is called only for sells that are select-able (which means they do not have interactive elements in them), that should basically change nothing for the cell.

Reply:
UPD: I changed that logic in this commit: https://github.com/cerner/terra-framework/pull/1950/commits/75f1f241355ea225b2b4cfb4ef22ac221df279d3

The logic is needed to prevent the click on the cell that has a button in it (the button should be clicked, but not the cell that contains it)

Here is how it works now:
`if (isSelectableCell) {
      if (onCellSelect) {
        onCellSelect({ rowId, columnId: id });
      }
    } else {
      // allows clickable elements inside non-selactable cells to be clicked, but not the cell itself
      event.preventDefault();
    }
  };`